### DISCUSSION TITLE
Account not visible to others

### DISCUSSION THREAD

**Discussion by iatomic1** [community/community#106594] [Other Feature Feedback, Questions, & Ideas]
I need someone to assist with giving information on the status of ticket #2564935. After I generated a new password, found out my account is flagged and when other people try to open it they get 404. The ticket is open for more than 7 days and it says, and it has passed 7days now, about a week and some days '...it may take up to seven business days to hear back from our Support Engineers.', so I'm a bit worried if it got lost somewhere down the line. Will appreciate any feedback - Thank you!
my original profile is @adeyemialameen04 https://github.com/adeyemialameen04

**Comment by wpritom** [comment]
I am facing the same issue. I can log into my account. But it is not visible to others. 

I've checked the settings page. The account is public. I noticed the issue first while trying to add another user as a contributor in one of my public repo. 

Even the public repo links are not found (404). 

Is there any way to solve the issue?
